model/usersModel.ts

```diff
@@ -0,0 +1,16 @@
+export const joinQuery = ({
+  uid,
+  nickname,
+  myTeam,
+}: {
+  uid: string;
+  nickname: string;
+  myTeam: string;
+}): [string, any[]] => {
+  const sql = `
+        INSERT INTO "BaseballArchive".users (uid, nickname, my_team) 
```

Comment:
이렇게 사용할 수 있게끔 하는건 불가능할까요?
```suggestion
        INSERT INTO users (uid, nickname, my_team) 
```

Reply:
postgersql을 처음 사용하는데 pgadmin에서는 그냥 users만 쓰면 실행이 안되더라구요!

Reply:
혹시 에러메시지는 어떻게 나왔을까요?
확인해봤을 때 옵션을 설정해주면 테이블 이름만 사용할 수 있는 것 같기도 해서요!

https://chatgpt.com/share/3f5761a2-0398-4ffa-a35e-c1965b52a931

Reply:
이 부분은 배포된 db로 연동하니 해결되었습니다.